### PR TITLE
[FR] Improve documentation

### PR DESCRIPTION
**Pull request by maririos**
Part of UX study feedback:
- Add to documentation description about what SAS Url is – link to service docs about training (we might already do this)
- Add readme code snippet pointers for statements in key concepts.
- For Copy, hard to find resource id and region. Add docstrings to clarify this more. Add in the documentation how/where to get the region for Copy

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs [hunk 1]
```diff
@@ -352,11 +354,13 @@ public virtual async Task<CopyModelOperation> StartCopyModelAsync(string modelId
         /// <summary>
         /// Generate authorization for copying a custom model into the target Form Recognizer resource.
         /// </summary>
-        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.</param>
-        /// <param name="resourceRegion">Location of the target Form Recognizer resource</param>
+        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.
+        /// This information can be found in the Properties section of the Form Recognizer resource in the Azure Portal.</param>
+        /// <param name="resourceRegion">Location of the target Form Recognizer resource.
+        /// This information can be found in the Properties section of the Form Recognizer resource in the Azure Portal.</param>
```
Comment (maririos) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: @kristapratico couldn't find the link for all the available regions. Do you have it?

**Review by kinelski — APPROVED** [review]
LGTM

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: https://azure.microsoft.com/global-infrastructure/services/?products=cognitive-services

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: Thank you.

Comment (jeremymeng) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: `resourceRegion` should be the `Location`  from the Keys section? Or the format `West US 2` also works?

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: Just saw this. sorry!!hmm yeah, better to say the Locaiton in Keys. I will update that in a future PR
thanks for pointing that out Jeremy!

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:360]: PR https://github.com/Azure/azure-sdk-for-net/pull/13096

**Pull request merged**